src/app/core.ts

```diff
@@ -8,14 +8,14 @@ import {
   randomUUID,
   throwError
 } from '../helpers/helpers';
-import {asperaDesktop} from '../index';
-import {DesktopInfo, TransferResponse} from '../models/aspera-desktop.model';
+import {asperaBrowser} from '../index';
```

Comment:
No expert but we probably don't want to change all these names in this file... This code is probably still Desktop App specific...

Reply:
This will match the naming of the SDK.  Ideally if Gateway is merged in here eventually these core services would all be used on the same thing per #39